_data/training-modules.yaml

```diff
@@ -164,3 +164,10 @@
   videos: ''
   webpage: https://hsf-training.github.io/hsf-training-ssh-webpage/
   id: ssh
+- description: A guide through the basics of the file systems and the shell.
+  name: UNIX
```

Comment:
I think `The Unix Shell` would be better here. This is what becomes the title, and unix is a biiiiig topic ;)